### PR TITLE
Add cache-from support

### PR DESCRIPTION
**Pull request by 72636c**
Related blogerature:

- https://medium.com/@gajus/making-docker-in-docker-builds-x2-faster-using-docker-cache-from-option-c01febd8ef84
- https://andrewlock.net/caching-docker-layers-on-serverless-build-hosts-with-multi-stage-builds---target,-and---cache-from/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,20 @@ steps:
...
+Cache](https://github.com/seek-oss/docker-ecr-cache-buildkite-plugin) plugin to
+reuse a cached base image that is shared with other pipeline steps:
+
+```yaml
+steps:
+  - plugins:
+      - seek-oss/docker-ecr-cache#v1.1.1:
+          target: deps
+      - seek-oss/docker-ecr-publish#v2.0.0:
+          cache-from: ecr://build-cache/org/my-repo
```
Comment (72636c) [inline README.md]: Hacked-together shorthand for a same-account repository; the underlying URL is retrieved by the plugin. I'm not sure if the `ecr://` prefix is appropriate.

Comment (bennycao) [reply README.md]: how about just `cache`

**Review by bennycao — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (72636c) [reply README.md]: Idea is to use the same name as the [`docker build` option](https://docs.docker.com/engine/reference/commandline/build/#options).

Comment (72636c) [reply README.md]: The `ecr://` substitution shouldn't break anything given that the underlying Docker command does not expect a protocol:

> [A registry path is similar to a URL, but does not contain a protocol specifier (https://).](https://docs.docker.com/engine/reference/commandline/pull/#pull-from-a-different-registry)

**Pull request merged**